client/question/components/LearnerViewDirective.js

```diff
@@ -651,6 +651,13 @@ tie.directive('learnerView', [function() {
          */
         $scope.printingIsSupported = PrintTerminalService.isPrintingSupported();
 
+        /**
+         * Defines whether TIE is currently being framed by the expected
+         * parent origin. If it is, the "Submit Code" button should be
+         * displayed.
+         */
+        $scope.isIframed = ParentPageService.isIframed();
```

Comment:
Here, we have things that are named `isIframed` being both a boolean and a function. This is a little confusing.

Perhaps you could say $scope.pageIsIframed instead to indicate that it's a boolean. Otherwise it feels like there should be parentheses at the end of that name.